changelog/change_add_documentationbaseurl.md

```diff
@@ -0,0 +1 @@
+* [#756](https://github.com/rubocop/rubocop-rails/pull/756): Add DocumentationBaseURL. ([@r7kamura][])
```

Comment:
How about the changelog entry for users like this:

```suggestion
* [#756](https://github.com/rubocop/rubocop-rails/pull/756): Fix incorrect documentation URLs when using `rubocop --show-docs-url`. ([@r7kamura][])
```

And I think `changelog/fix_add_documentationbaseurl.md` is more appropriate than `changelog/change_add_documentationbaseurl.md` as the file name.

Reply:
Good point 🦀 

I'll improve them also in the other related pull requests